docs/add-ins/addin-tutorial.md

```diff
@@ -1,107 +1,219 @@
-    - If your browser indicates that the site's certificate is not trusted, you will need to add the certificate as a trusted certificate. Outlook will not load add-ins if the site is not trusted. See [Adding Self-Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) for details.
-    - If the browser does not indicate a problem with the certificate, proceed to the next step.
-1. Sideload the add-in using the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md). When browsing for the manifest file, select `manifest.xml` in the project root.
-1. You should now see a new button on the ribbon labeled **Display all properties**. Click this button to open the task pane. You should see the add-in's welcome page.
+        ```HTML
+        <html>
+          <head>
+            <style>
+            h1 {
+              font-family: Calibri;
+            }
+            </style>      
+          </head>
+          <body>
+            <h1>Hello World!</h1>
+            <p>This is a test</p>
+          </body>
+        </html>
+        ```
 
-    ![A screenshot of the button and task pane added by the sample](images/addin-tutorial/button-and-pane.PNG)
+    - Select the **Create public gist** button.
 
-## Write the code
+## Create an Outlook add-in project
 
-Now that we've verified that the base add-in works, we can customize it to do what we want. We'll start with the manifest.
+Use the Yeoman generator to create an Outlook add-in project.
 
-### Update the manifest (again)
+1. Run the following command via the command prompt and then answer the prompts as follows:
 
-The manifest for an add-in controls how it appears in Outlook. It defines the way the add-in appears in the add-in list, the buttons that appear on the ribbon, and it sets the URLs for the HTML and JavaScript files used by the add-in.
+    ```bash
+    yo office
+    ```
+
+    - **Choose a project type** - `Office Add-in project using Jquery framework`
+    - **Choose a script type** - `Javascript`
+    - **What do you want to name your add-in?** - `Git the gist`
+    - **Which Office client application would you like to support?** - `Outlook`
+
+    ![A screenshot of the prompts and answers for the Yeoman generator](images/addin-tutorial/yeoman-prompts.PNG)
+    
+    After you complete the wizard, the generator will create the project and install supporting Node components.
+	
+1. Navigate to the root directory of the project.
 
-Let's start by updating some properties of the add-in itself.
+    ```bash
+    cd "Git the gist"
+    ```
+
+1. This add-in will use the [Showdown](https://github.com/showdownjs/showdown) library to convert Markdown to HTML and the [URI.js](https://github.com/medialize/URI.js) library to build relative URLs. To install these tools for your project, run the following command in the root directory of the project:
```

Comment:
Hmmmm...I see your point in general and keep this guidance in mind. In this case though, the add-in doesn't yet use the Showdown and URI.js libraries (because the user hasn't added that functionality yet), so present tense isn't really appropriate. The add-in *will* use those libraries, after the user adds the functionality that comes later in the tutorial.